locale/pt_PT.ts

```diff
@@ -0,0 +1,5213 @@
+        <location filename="../language.cc" line="169"/>
+        <source>Nepali</source>
+        <translation>Nepalese</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="170"/>
+        <source>Ndonga</source>
+        <translation>Ndonga</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="171"/>
+        <source>Dutch</source>
+        <translation>Holandês</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="172"/>
+        <source>Norwegian Nynorsk</source>
+        <translation>Norueguês</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="173"/>
+        <source>Norwegian</source>
+        <translation>Norueguês</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="174"/>
+        <source>South Ndebele</source>
+        <translation>Ndebele do Sul</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="175"/>
+        <source>Navajo</source>
+        <translation>Navajo</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="176"/>
+        <source>Chichewa</source>
+        <translation>Chichewa</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="177"/>
+        <source>Occitan</source>
+        <translation>Occitano</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="178"/>
+        <source>Ojibwa</source>
+        <translation>Ojibwa</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="179"/>
+        <source>Oromo</source>
+        <translation>Oromo</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="180"/>
+        <source>Oriya</source>
+        <translation>Oriya</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="181"/>
+        <source>Ossetian</source>
+        <translation>Osseto</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="182"/>
+        <source>Panjabi</source>
+        <translation>Panjabi</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="183"/>
+        <source>Pali</source>
+        <translation>Pali</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="184"/>
+        <source>Polish</source>
+        <translation>Polonês</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="185"/>
+        <source>Pashto</source>
+        <translation>Pashtu</translation>
+    </message>
+    <message>
+        <location filename="../language.cc" line="186"/>
+        <source>Portuguese</source>
+        <translation>Português brasileiro</translation>
```

Comment:
This translation is wrong both here and in the source file *locale/pt_BR.ts*. Here how it looks in the GoldenDict UI:
![portuguese-wrong-lang-translation](https://github.com/user-attachments/assets/ba0af088-0148-4abf-8dda-8402c0827e08)

The flag is enough to disambiguate the language variants. And that's how it works for other languages. Would the single word `Português` be correct for both Portuguese language variants?